src/App/Services/Form.php

```diff
@@ -220,13 +220,11 @@ public function sectionVisibility($fields, bool $hidden): self
         return $this;
     }
 
-    public function tabVisibility($tabs, $hidden): self
+    public function tabVisibility($tabs, bool $hidden): self
     {
-        $this->template->get('sections')
-            ->filter(fn ($section) => (new Collection($tabs))
-                ->contains($section->get('tab'))
-            )->each(fn ($section) => $section->get('fields')
```

Comment:
since filter returns a new collection instance this did not alter the original objects

Reply:
Good point but we can have multiple sections under one tab